src/blob_file_set.cc

```diff
@@ -94,6 +94,12 @@ Status BlobFileSet::Recover() {
                    "Next blob file number is %" PRIu64 ".", next_file_number);
   }
 
+  // Make sure perform gc on all files at the beginning
+  MarkAllFilesForGC();
```

Comment:
Just curious. Why the call needs to move up here?

Reply:
oh, no need actually. I forget to revert it.